fission/src/ui/modals/configuring/SettingsModal.tsx

```diff
@@ -83,159 +88,189 @@ const SettingsModal: React.FC<ModalPropsImpl> = ({ modalId }) => {
         PreferencesSystem.savePreferences()
     }
 
+    const resetGeneral = () => {
+        setReportAnalytics(PreferencesSystem.getGlobalPreference("ReportAnalytics"))
+        setRenderScoringZones(PreferencesSystem.getGlobalPreference("RenderScoringZones"))
+        setRenderSceneTags(PreferencesSystem.getGlobalPreference("RenderSceneTags"))
+        setRenderScoreboard(PreferencesSystem.getGlobalPreference("RenderScoreboard"))
+        setSubsystemGravity(PreferencesSystem.getGlobalPreference("SubsystemGravity"))
+        setShowViewCube(PreferencesSystem.getGlobalPreference("ShowViewCube"))
+        setMuteAllSound(PreferencesSystem.getGlobalPreference("MuteAllSound"))
+        setSFXVolume(PreferencesSystem.getGlobalPreference("SFXVolume"))
+        setSceneRotationSensitivity(PreferencesSystem.getGlobalPreference("SceneRotationSensitivity"))
+        setViewCubeRotationSensitivity(
+            PreferencesSystem.getGlobalPreference<number>("ViewCubeRotationSensitivity") * 60
+        )
+    }
+
+    const saveHandlers: Record<"general" | "graphics", () => void> = {
+        general: saveSettings,
+        graphics: () => graphicsRef.current?.save()!,
+    }
+    const resetHandlers: Record<"general" | "graphics", () => void> = {
+        general: resetGeneral,
+        graphics: () => graphicsRef.current?.reset()!,
```

Comment:
Both of these chains are unnecessary, you would only need them if you were passing in the potentially undefined `save` method itself.
```suggestion
        graphics: () => graphicsRef.current?.save(),
    }
    const resetHandlers: Record<"general" | "graphics", () => void> = {
        general: resetGeneral,
        graphics: () => graphicsRef.current?.reset(),
```